build/clean.js

```diff
@@ -15,10 +15,20 @@ const paths = argv._;
  * Function to remove a given path
  */
 function cleanPath(cleanPath) {
-    const removePath = path.resolve(process.cwd(), cleanPath)
-    rimraf(removePath, () => {
-        console.log(removePath, "cleaned");
-    });
+    if (fs.existsSync(path)) {
+        fs.readdirSync(path).forEach(function(file, index){
+            var currentPath = `${path}/${file}`;
+            
+            if (fs.lstatSync(currentPath).isDirectory()) { // recurse
+                deleteFolderRecursive(currentPath);
```

Comment:
where is this coming from? Maybe just leave rimraf if you're gonna have to write all of this.